cns/configuration/configuration.go

```diff
@@ -63,6 +63,8 @@ type TelemetrySettings struct {
 	DebugMode bool
 	// Interval for sending snapshot events.
 	SnapshotIntervalInMins int
+	// AppInsightsInstrumentationKey allows the user to override the default appinsights ikey
```

Comment:
Typo?
```suggestion
	// AppInsightsInstrumentationKey allows the user to override the default appinsights key
```

Reply:
It was intentional though I can see why it seems that way. 😄  I've seen appinsights instrumentation keys abbreviated as "ikey" which is what I was going for here.